buf/registry/module/v1beta1/label.proto

```diff
@@ -71,6 +72,30 @@ message Label {
     (buf.validate.field).required = true,
     (buf.validate.field).string.uuid = true
   ];
+
+  // Whether or not governance review is enabled for this Label.
+  //
+  // Governance is an enterprise-only feature - contact us to learn more!
+  bool review_enabled = 10;
+  // The current review state of the Label.
+  //
+  // This field will only be set if governance is enabled.
+  LabelReviewState review_state = 11 [(buf.validate.field).enum.defined_only = true];
```

Comment:
I don't understand what this field is for. Is this the review state of the commit pointed to by `commit_id`? If so, that can never be `_REJECTED`.

Is this the review state of the most recent commit on the label (Which may be any state, but may be the state of a commit that is different than the commit pointed to by `commit_id`)?

Reply:
In other words, review state is only a property of a Label plus a Commit, but since this field is on a Label, it is unclear which Commit the review state applies to

Reply:
I see this was answered below in the docs on LabelReviewState. My bad.